src/plugin/cesium/mixin/cesiummixin.js

```diff
@@ -8,6 +8,31 @@ goog.require('os.MapEvent');
 goog.require('os.net.Request');
 
 
+/**
+ * Gets the port from a URL. Cesium requires passing a valid port to its TrustedServers API.
+ * Returns 443 for https, 80 for http and everything else that isn't specified.
+ * @param {URL} url The URL to get the port from.
+ * @return {number} The port.
+ */
+const getPort = (url) => {
```

Comment:
Only `goog.module` files should declare things at the file scope. Outside of a module, you're declaring `window.getPort` to be this function.